etc/spack/defaults/windows/config.yaml

```diff
@@ -3,3 +3,5 @@ config:
   concretizer: clingo
   build_stage::
     - '$spack/.staging'
+    - 'C:/spack'
```

Comment:
I think this should not be added as a secondary option:

* This list is if earlier entries are inaccessible (i.e. permissions failures)
* I wanted to avoid using this directory without the user's knowledge
* If the prior point is overruled, this should replace the first entry (else the user would still encounter path-length failures).